lib/Doctrine/DBAL/Platforms/DB2Platform.php

```diff
@@ -97,7 +98,7 @@ public function initializeDoctrineTypeMappings()
      */
     public function isCommentedDoctrineType(Type $doctrineType)
     {
-        if ($doctrineType->getName() === Type::BOOLEAN) {
+        if ($doctrineType->getName() === DefaultTypes::BOOLEAN) {
```

Comment:
The name `DefaultTypes` (as well as `BuiltinTypes`) feels weird in this context. It creates an impression that the code works only with default types but not with custom. Would it make sense to rename it to just `Types`? It's irrelevant whether they are default, built-in or custom in most contexts.

It may be hard to come up with a decent name because this class doesn't seem to have any purpose. What exactly is the reason for extraction?

Reply:
I'm fine with `Types`, but it may cause confusion that there are only these types and no other custom ones.

> What exactly is the reason for extraction?

- We extracted constants elsewhere into constant-only classes, but not here.
- In order to make Type an interface, we need to do a cleanup of the interface and these names don't belong there.

Reply:
Renamed to `Types`.

> What exactly is the reason for extraction?

Did I sufficiently answer your question in my previous comment?

Reply:
Yes, thank you.